### ISSUE TITLE
Angular-Meteor Seed Project

### ISSUE THREAD

**Issue by louiealmeda** [Urigo/angular-meteor#1325]
Possibly the best way to make more people jump-in to this more easily is to have a seed project that will:

- [ ] implement a simple working project (probably the todo app with user logins, with privacy of items)
- [ ] follows John Papa's style guide?
- [ ] implement `angular 2 Now` for front end
- [ ] implement angular server
- [ ] structured project directory
- [ ] pre-configured meteor app (Blaze and other unnecessary things removed, angular added, etc)

I think `meteor-angular-socially` do not qualify as a seed project because it does not implement a structured project directory the way you recommend it (am I right?)

Basically, everything would be bootstrapped, all that is left is for the developer to add in the features that he needs to. Having this seed project more users can jump-in and start exploring this platform easily. It is a project that you can base on to, which doubles as a playground.

This will also be a great way to showcase your angular-meteor full stack platform

I've searched the web for some time now and not seen any sign of this seed project. 
Thanks for the great work, looking forward for this project to come.

**Comment by Tallyb** [comment]
As of now - this is probably the best place to start: 
https://www.meteor.com/tutorials/angular/creating-an-app

**Comment by Urigo** [comment]
We've updated Socially to the best practices for Meteor 1.3 and Angular 1.5.
I think it's a good seed project - https://github.com/Urigo/meteor-angular-socially
